ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/switch/v1/switch/README.md

```diff
@@ -0,0 +1,79 @@
+<!--
+Copyright 2023 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form Switch (v1)
+====
+Adaptive Form Switch component written in HTL.
+
+## Features
+
+* Provides the following type of input:
+  * switch
+* Custom constraint messages for the above types
+* Styles
+
+### Use Object
+The Form switch component uses the `com.adobe.cq.forms.core.components.models.form.Switch` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Form Switch component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this field
+2. `./hideTitle` - if set to `true`, the label of this field will be hidden
+3. `./name` - defines the name of the field, which will be submitted with the form data
+4. `./description` - defines a help message that can be rendered in the field as a hint for the user
+5. `./readOnly` - if set to `true`, the filed will be read only
+6. `./type` - defines the data type of the value
+7. `./enum` - defines the two set of possible values for this field. (On or Off)
+
+## Enum
+Switch component can have only one of two state i.e. ON and OFF. Since the enums are stored as array, the enum at index '0' is assumed to be OFF(i.e. switch is 'off'), and enum at index '1' is ON (i.e. switch is 'on'')
+
+## Client Libraries
+The component provides a `core.forms.components.switch.v1.runtime` client library category that contains the Javascript runtime for the component. 
+It should be added to a relevant site client library using the `embed` property.
+
+It also provides a `core.forms.components.switch.v1.editor` editor client library category that includes
+JavaScript handling for dialog interaction. It is already included by its edit dialog.
+
+## BEM Description
+```
+BLOCK cmp-adaptiveform-switch
```

Comment:
Please update Readme.md with all BEM blocks. For example, widget-checkbox is not present and it is documented here. I see some BEM like `cmp-adaptiveform-switch__circle` in the sightly but it is not mentioned here